contactez-nous/contactez-nous.md

```diff
@@ -0,0 +1,103 @@
+---
+altLangPage: "#"
+breadcrumbs:
+  - title: À propos de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: Système de conception de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+css:
+- https://design.canada.ca/css/split-h1.css
+- https://design.canada.ca/css/custom.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+date: 2023-03-20
+dateModified: 2023-03-20
+description: "Contactez le Bureau de la transformation numérique"
+lang: en
+layout: void
+title: "Contactez le Bureau de la transformation numérique"
+---
+<h1 property="name" id="wb-cont" dir="ltr"><span class="stacked"><span>Contactez le Bureau de la transformation numérique</span>: <span>Canada.ca design</span></span></h1>
```

Comment:
The default template will implement the stacked h1 and the gc-thickline by default once you rebase.